packages/kbn-optimizer/limits.yml

```diff
@@ -69,7 +69,7 @@ pageLoadAssetSize:
   searchprofiler: 67080
   security: 189428
   securityOss: 30806
-  securitySolution: 283440
+  securitySolution: 235402
```

Comment:
@tylersmalley Hey! Thanks for chatting with me earlier. I've updated the `securitySolution` limits here and am investigating how to decrease the lists plugin size.